templates/zerver/integrations.html

```diff
@@ -1106,6 +1112,42 @@
       <img class="screenshot" src="/static/images/integrations/jira/001.png" />
     </div>
 
+   <div id="librato" class="integration-instructions">
+
+    <p>Get Zulip notifications for your Librato alerts!</p>
+
+    <p>First, create the stream you'd like to use for Librato notifications, and
+    subscribe all interested parties to this stream. We recommend the
+    name <code>librato</code>.</p>
+
+    <p>Next, on your <a href="/#settings" target="_blank">Zulip settings
+    page</a>, create an Librato bot. Please note the bot name and API key. Then:</p>
+
+      <p>1. Go to your integrations page:</p>
+      <p><img src="/static/images/integrations/librato/001.png" /></p>
+
+      <p>2. From there, select <b>"Webhook"</b> integration:</p>
+      <p><img src="/static/images/integrations/librato/002.png" /></p>
+
+      <p>3. Fill the title and URL fields using your bot api-key</p>
+      <p><img class="screenshot" src="/static/images/integrations/librato/003.png" /></p>
```

Comment:
I think it might be a good idea to retake this screenshot with a clearly fake API key like we use in the other screenshots...